typescript/cli/src/utils/keys.ts

```diff
@@ -77,14 +78,14 @@ async function addressToImpersonatedSigner(
  * @param key a private key
  * @returns a signer for the private key
  */
-function privateKeyToSigner(key: string): ethers.Wallet {
+function privateKeyToSigner(key: string): Wallet {
```

Comment:
so we can use zksync ethers here and it'll still be backwards compatible with non-zksync chains?

Reply:
According to the testing that we have done so far, zksync wallet is fully compatible with ethers.Wallet. Also, in the `getContext` function, the function for getting signer is called before MultiProvider is instantiated, so there is not way of passing `ProtocolType` to the getSigner function.

For now this works, but in the future it would require heavier refactoring for supporting different types of signers per protocol.